src/Kalkulator/kalkulator-utils.ts

```diff
@@ -0,0 +1,21 @@
+import { Sykefraværshistorikk, SykefraværshistorikkType } from '../api/sykefraværshistorikk';
+
+const summer = (tall: number[]) => tall.reduce((a, b) => a + b);
+
+export const getAntallTapteDagsverkSiste4Kvartaler = (
+    historikkListe: Sykefraværshistorikk[]
+): number | 'erMaskertEllerHarIkkeNokData' => {
+    const alleProsenter = [
+        ...historikkListe.find(historikk => historikk.type === SykefraværshistorikkType.VIRKSOMHET)!
```

Comment:
Får jeg lov til å spørre om hva gjør den "!"

Reply:
Det er en null/undefined-sjekk :) `a!` gir en error hvis a er undefined eller null. Jeg bruker det her fordi vi _vet_ at det finnes statistikk for virksomhet, selv om det ikke kommer fram i typene våre.